framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -133,6 +134,9 @@ public ValueAnnotatedTypeFactory(BaseTypeChecker checker) {
         // this line just registers the alias. The BottomVal is never used.
         addAliasedAnnotation(MinLen.class, BOTTOMVAL);
 
+        addAliasedAnnotation(
+                "org.checkerframework.checker.index.qual.Positive", createIntRangeFromPositive());
```

Comment:
None of those allow us to derive a `@MinLen` annotation. We could do something similar for `@NonNegative`, etc., but Mike and I discussed it and decided to just do `@Positive` first, and then we could copy whatever approach we used there. I should have explained that decision in the PR.